### DISCUSSION TITLE
orderBy using case

### DISCUSSION THREAD

**Discussion by rahmat-dev** [prisma/prisma#19824] [Q&A] [answered]
Can I order data like this query without using the raw method?

```sql
SELECT * FROM your_table
    ORDER BY
        CASE
            WHEN condition THEN 1
            ELSE 0
        END ASC,
        createdAt DESC;
```

**Comment by ludralph** [comment, marked as answer]
Hi @rahmat-dev 👋 

Thank you for raising this question. Currently, this is not supported, so you would need to use query raw to achieve this. However, we do have a feature request to support this behavior.

- https://github.com/prisma/prisma/issues/10335

Do add a 👍 to the issue to help our engineers prioritize it while also documenting your use case.


If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by rahmat-dev** [reply]
Thank you for the information. I hope it will be realized soon
